src/scss/components/_rate.scss

```diff
@@ -0,0 +1,73 @@
+$rate-color: $grey-lighter !default;
+$rate-color-active: darken($warning, 5%) !default;
+$rate-space: 5px !default;
+
+.rate {
+  align-items: center;
+  display: flex;
+  &:not(:last-child) {
+    margin-bottom: ($block-spacing / 2);
+  }
+  &.is-spaced .rate-item:not(:last-child) {
+    margin-right: $rate-space;
+  }
+  &.is-disabled .rate-item {
+    cursor: initial;
+    &:hover {
+      transform: none;
+    }
+  }
+  .rate-item {
+    cursor: pointer;
+    display: inline-flex;
+    position: relative;
+    transition: all 0.3s;
+    &:hover {
+      transform: scale(1.1);
+    }
+    &.set-on .icon,
+    &.set-half .is-half {
+      color: $rate-color-active;
+    }
+  }
+  .icon {
+    color: $rate-color;
+    pointer-events: none;
+    width: inherit;
+    // fix MDI when half star in all size
+    line-height: 1;
+  }
+  .is-half > i {
```

Comment:
This part is quite tricky. What happen when the icon pack isn't FA o MDI ? Because in the next release (in the current dev code) there is a feature that allows to add new icon packs

Reply:
Actually not only for MDI, but other icons can. Because I can't try all font icons so this style is just to reset the line height.

If there is a problem about this later, then I will fix it again soon.

Reply:
Yes, but I mean `is-half` style and `icon` class override

Reply:
Yes about that, I think it is the best solution at this time to support decimal values, please add this PR, so that in the latest version I can make it better with the latest features on the component icons. and support for all font icons listed in bulma.

sorry if my english is not good.

Reply:
Buefy should accept other icon packs, but I think it should officially support only FA and MDI.
People who uses other packs should be responsible for style customization if needed.
So I think, it is probably good this way.